include/ROSNode.h

```diff
@@ -0,0 +1,37 @@
+/*
+	Copyright 2021 OpenJAUS, LLC (dba MechaSpin). Subject to the MIT license.
+*/
+
+#pragma once
```

Comment:
Possibly an errant #pragma once

Reply:
Hmm. Seems a missed these on the ROS and Qt app. I'll fix them real quick.